test/new-e2e/runner/configmap.go

```diff
@@ -10,10 +10,20 @@ import (
 	"errors"
 
 	"github.com/DataDog/datadog-agent/test/new-e2e/runner/parameters"
+	infraaws "github.com/DataDog/test-infra-definitions/aws"
+	commonconfig "github.com/DataDog/test-infra-definitions/common/config"
 
 	"github.com/pulumi/pulumi/sdk/v3/go/auto"
 )
 
+const (
+	AgentAPIKey               = commonconfig.DDAgentConfigNamespace + ":" + commonconfig.DDAgentAPIKeyParamName
```

Comment:
Can you remove the file https://github.com/DataDog/datadog-agent/blob/0323cd2a0cc0f14a1d6f9822b94e97ef4d9459a5/test/new-e2e/runner/parameters/key_name.go#L15 which has the same purpose as the code you added?